provisioning/miniprov/hare_mp/main.py

```diff
@@ -53,12 +59,32 @@ def execute(cmd: List[str]) -> str:
     return out
 
 
-def setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+def create_logger_directory():
+    """Create log directory if not exsists."""
```

Comment:
```suggestion
    """Create log directory if not exsits."""
```

Reply:
done

Reply:
@supriyachavan4398, did you address this comment? I don't see the code change.

Reply:
Addressed